dts/arm64/qemu/qemu-virt-a53.dtsi

```diff
@@ -91,5 +100,36 @@
 			 */
 			reg = <0x0 0x0 0x0 DT_SIZE_M(64) /* 0x4000000 DT_SIZE_M(64) */>;
 		};
+
+		pcie: pcie@4010000000 {
+			compatible = "pci-host-ecam-generic";
+			device_type = "pci";
+			reg = <0x40 0x10000000 0x00 0x10000000>;
+			#size-cells = <0x02>;
+			#address-cells = <0x03>;
+			ranges = <0x1000000 0x00 0x00        0x00 0x3eff0000  0x00 0x10000
+				  0x2000000 0x00 0x10000000  0x00 0x10000000  0x00 0x2eff0000
+				  0x3000000 0x80 0x00        0x80 0x00        0x80 0x00>;
+			#interrupt-cells = <0x01>;
+			interrupt-map-mask = <0x1800 0x00 0x00 0x07>;
+			interrupt-map = <0x00   0x00 0x00 0x01 0x8002 0x00 0x00 0x00 0x03 0x04
```

Comment:
Could this part be much more readable? For example, replace some numbers with macro. Just a suggestion and I won't block this PR from being merged.

Reply:
Hello SgrrZhf. Thank you for the idea.

I have updated the definition with a new commit. If you could please take a look.

I would like to use `phandle-array` (which seems to me the right choice for the property type); but we could not make it work. So we looked at some other examples (gpio-map), and they were using the `compound` type (`dts/bindings/gpio/gpio-nexus.yaml`). So that is the solution we used here.

The only way we could make it work with `phandle-array` was adding `#` for `interrupt-map`.. but this seems a hack.

I also had to swap some hexadecimal values for decimal so the line would fit in 100 characters (compliance).